### PR TITLE
Add new logger module to use for reporting to kibana

### LINKED ISSUE #585: Post to kibana when executing workflow-job which executes an ensemble
We are deprecating the workflow-jobs which executes and ensemble. These jobs has been used for running a series of simulations, which now is possible with using bash-script and the CLI. 

We should report  back to kibana who is actually using this, in order to show them the new path, and lead them into enlightenment.

### PR DESCRIPTION
**Pull request by ManInFez**
Use new module in network-reporting in job_runner

Report use deprecated WF jobs to kibana

resolves: https://github.com/equinor/ert/issues/585

### CODE REVIEW

**Review by sondreso — APPROVED** [review]
LGTM

File: python/ert_logger/__init__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        proxies = {"http": None, "https": None}
+        requests.post(
+            LOG_URL,
+            timeout=3,
+            headers={"Content-Type": "application/json"},
+            data=data,
+            proxies=proxies,
+        )
+    except:  # noqa
+        pass
```
Comment (mortalisk) [inline python/ert_logger/__init__.py:29]: do we want to pass here?

Comment (jondequinor) [reply python/ert_logger/__init__.py:29]: Not really, but hard to know, but if you catch anything, you need to make sure you've caught them all. Unless—this code will break `job_dispatch`.

**Review by jondequinor — APPROVED** [review]
LGTM

**Pull request merged**